ui/js/graphs.js

```diff
@@ -423,22 +423,33 @@ perf.controller('GraphsCtrl', [
     }
 
     $scope.repoName = $stateParams.projectId;
-
+    
     function updateDocument() {
       $state.transitionTo('graphs', {
         timerange: $scope.myTimerange.value,
         series: $scope.seriesList.map(function(series) {
-          return encodeURIComponent(
-            JSON.stringify(
-              { project: series.projectName,
-                signature: series.signature,
-                visible: series.visible})); }),
+               var returnSeries = "[" + series.projectName + "," + series.signature + "," + ((series.visible == true)? 1:0) + "]"
+          return returnSeries}),
```

Comment:
no need for the returnSeries temporary variable, just return the string

Also change: ((series.visible == true)? 1:0)  --> (series.visible ? 1 : 0); (note spacing changes as well)